src/Coordination/CoordinationSettings.h

```diff
@@ -48,7 +48,8 @@ struct Settings;
     M(UInt64, configuration_change_tries_count, 20, "How many times we will try to apply configuration change (add/remove server) to the cluster", 0) \
     M(UInt64, max_log_file_size, 50 * 1024 * 1024, "Max size of the Raft log file. If possible, each created log file will preallocate this amount of bytes on disk. Set to 0 to disable the limit", 0) \
     M(UInt64, log_file_overallocate_size, 50 * 1024 * 1024, "If max_log_file_size is not set to 0, this value will be added to it for preallocating bytes on disk. If a log record is larger than this value, it could lead to uncaught out-of-space issues so a larger value is preferred", 0) \
-    M(UInt64, min_request_size_for_cache, 50 * 1024, "Minimal size of the request to cache the deserialization result. Caching can have negative effect on latency for smaller requests, set to 0 to disable", 0)
+    M(UInt64, min_request_size_for_cache, 50 * 1024, "Minimal size of the request to cache the deserialization result. Caching can have negative effect on latency for smaller requests, set to 0 to disable", 0) \
+    M(UInt64, raft_limits_reconnect_limit, 50, "If connection to a peer is silent longer than this limit * (multiplied by heartbeat interval), we re-establish the connection.", 0)
```

Comment:
seems that the word 'limit' means 'threshold'

Reply:
Sorry for late reply. Maybe my English is not good enough, so please ignore my comment if I am wrong.

I just feel 'limit' means 'cannot exceeds it'; 'threshold' means 'if the threashold is reached, then do something'. 

This setting means 'if silent time reach to the number of the setting, then reconnect.', so I feel it is threshold.

Reply:
Yes you would be technically correct and that's a nice observation.
I would leave it as is so the mapping to NuRaft is more obvious and it would be a bit more complex and confusing to introduce the same setting with a different name.